### PR TITLE
Finish the currently existing lore pages

### LINKED ISSUE #372: finish mock character / lore pages

### PR DESCRIPTION
**Pull request by matr1x-hackmud**
### Problem
The wiki pages currently under /lore consist of 2 documents:

- characters/robovac_catesby
- events/robovac_revolution_v1

These were initially put up as placeholders and have changed very little since their initial commits. These ought to be completed prior to https://github.com/comcode-org/hackmud_wiki/issues/371 and the completion of P009

Fixes #372

### Context

This PR also checks in updates to the biography and event templates. Please have a look at them and let me know if you have any critiques or questions

### CODE REVIEW

File: docs/lore/characters/robovac_catesby.mdx [hunk 1]
```diff
@@ -2,17 +2,29 @@
...
+
+robovac_catesby's name, similar to other members of the Thirteen, derives from [Robert Catesby](https://en.wikipedia.org/wiki/Robert_Catesby) - the leader of the failed Gunpowder Plot of 1605 AD.
 
 ## Appears in
 
-robovac_revolution_v1
-ministry of dust
-parade
-parade 2
+- [robovac_revolution_v1](../events/robovac_revolution_v1)
```
Comment (seanmakesgames) [inline docs/lore/characters/robovac_catesby.mdx:26]: this should be a wikilink, right? (I don't remember the syntax)

File: docs/lore/characters/robovac_catesby.mdx [hunk 2]
```diff
@@ -2,17 +2,29 @@
...
+- robovac_revolution_v3
```
Comment (seanmakesgames) [inline docs/lore/characters/robovac_catesby.mdx:27]: These could be 'dead' wikilinks -- don't remember what our policy is right now.

File: docs/lore/events/robovac_revolution_v1.mdx [hunk 3]
```diff
@@ -2,14 +2,72 @@
...
+- robovac_percy
+- robovac_fawkes
+- robovac_keyes
+- robovac_bates
+- robovac_r_wintour
+- robovac_c_wright
+- robovac_grant
+- robovac_rookwood
+- robovac_digby
+- [robovac_catesby](../characters/robovac_catesby)
```
Comment (seanmakesgames) [inline docs/lore/events/robovac_revolution_v1.mdx:69]: wikilinks?

File: docs/lore/characters/robovac_catesby.mdx [hunk 4]
```diff
@@ -2,17 +2,29 @@
...
 
-robovac_catesby is a robovac. He seems to like teapots. He also flashes lights a lot.
+robovac_catesby is a robovac and member of 'The Thirteen'; a council of revolutionary robovacs. Despite his robovac-pattern, he fancies himself to be a teapot. He muses a lot on the nature of balance and equillibrium.
 
-## Brief Summary of character
+## Character Summary
 
-robovac_catesby was the ring leader for robovac_revolution_v1. While silent through the first, and missing in the second, revolution, he begins to take a more center stage appearance in later events. He becomes a tea pot and hosts some tea parties during the ministry of dust. I can't really remember much more.
+robovac_catesby has been present for several robovac revolutions, leading [the first](../events/robovac_revolution_v1). While in hiding throughout the first; and missing in the second; he returned for the third revolution, a changed vac. A message delivered by Faythe ('<span class="color-tag" data-tag="I">418</span>') has led him to believe that he is, in fact, a teapot. This has given him a unique perspective on Trust's domain and philosophy in general compared to some of his narrower-minded friends.
```
Comment (seanmakesgames) [inline docs/lore/characters/robovac_catesby.mdx:11]: wikilink?

**Review by seanmakesgames — APPROVED** [review]

File: docs/lore/characters/robovac_catesby.mdx (hunk 1 above)
Comment (matr1x-hackmud) [reply docs/lore/characters/robovac_catesby.mdx:26]: Correct. I also didn't remember the syntax, but I've gone ahead and updated all of these where applicable. I'll go do a walk across the other pages and check in a PR for any non-wikilinks that ought to be wikilinks.

**Pull request merged**